.gitmodules

```diff
@@ -40,7 +40,11 @@
   branch = develop
 [submodule "AQM"]
   path = AQM
-  url = https://github.com/NOAA-EMC/AQM
+  url = https://github.com/climbfuji/AQM
+  branch = feature/ufs-bundle-dom
+[submodule "NOAHMP"]
```

Comment:
Note. This PR contains develop as of Nov 14, therefore the NoahMP submodule still shows up as a change. Need to pull in/fast forward and then this should go away (and some other stuff)